K2Bridge/Factories/AggregateFactory.cs

```diff
@@ -138,6 +143,88 @@ public static void AddAggregates(this AggregateDictionary aggregateDictionary, s
             }
         }
 
+        /// <summary>
+        /// Get extended stats for standard deviation aggregate from a given <see cref="DataRow"/>.
+        /// </summary>
+        /// <param name="columnName">The column name.</param>
+        /// <param name="columnMetadata">The column metadata parsed.</param>
+        /// <param name="row">The row to be parsed.</param>
+        /// <param name="logger">ILogger object for logging.</param>
+        /// <returns><see cref="ExtendedStatsAggregate"></returns>
+        private static ExtendedStatsAggregate GetExtendedStatsAggregate(string columnName, string[] columnMetadata, DataRow row, ILogger logger)
+        {
+            logger.LogTrace("Get extended stats for standard deviation aggregate for {}", columnName);
+
+            var extendedStatsAggregate = new ExtendedStatsAggregate();
+
+            if (row[columnName] == DBNull.Value)
+            {
+                return extendedStatsAggregate;
+            }
+
+            var sigma = Convert.ToInt32(columnMetadata[2]);
```

Comment:
can we move '2' to an enum?

Reply:
same as columnName

Reply:
We have this index used in several aggregations so far.
I guess it could be a good idea to move it to an enum, but it should be part of a global PR to get something homogenous for all aggregations
your thought ?

Reply:
ok, please open an issue